packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js

```diff
@@ -30,15 +30,16 @@ function InputDateTimePicker(props) {
 					<div className={theme['date-time-picker']}>
 						<InputDatePicker
 							id={`${props.id}-date-picker`}
+							readOnly={props.readOnly}
+							onBlur={props.onBlur}
 							onChange={onDateChange}
 							value={date}
 							dateFormat={props.dateFormat}
-							useUTC={props.useUTC}
-							timezone={props.timezone}
-							hideTimezone
```

Comment:
we don't need to pass utc/timezone related props to InputDatePicker anymore, because utc/timezone will be handled on datetime level.